event/listener.php

```diff
@@ -62,6 +63,20 @@ public static function getSubscribedEvents()
 	}
 
 	/**
+	 * Adds the permission to the right permission category
+	 *
+	 * @param object $event The event object
+	 * @return void
+	 */
+	public function adv_polls_permissions($event)
```

Comment:
It is added to the "Misc" category, and cannot be changed any other way than by subscribing to this event...
This is something that should be stored in the DB, and changed in migration files, but... not there yet :-1: